### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by datagutten** [fizzwidget/feed-o-matic#10]
With WoW classic pets again need food to be happy, but blizzard has changed some functions causing this addon to stop working. I want to update it to make it work with classic, but then you have to add an open-source license to the repository. Can you do that?
I like this addon very much, especially the emotes like "Hey! Watch the fingers", and I hope to be able to use it for classic.

**Comment by beren12** [comment]
Do you know the changes? If so you could post them under a BSD or public domain license and it wouldn't matter what the original license is… but I agree open source would be best

**Comment by datagutten** [comment]
When there is no license specified it should be considered as "all rights reserved" and I am not allowed to make any changes without permission from the author.

**Comment by beren12** [comment]
Not quite. You are allowed to make all the changes you want, privately. You are not allowed to distribute a pre modified version. You are also free to post changes that others could use to patch. This hasn’t stopped fan updates of popular addons though. Hard to stop people anyway when blizz keeps breaking things and the source is a script language.

**Comment by ShaunaGordon** [comment]
Fizz has been MIA for a while now, unfortunately. I tried pinging him a couple of months back and never got a response. :(

That said, he moved the source for his stuff to Github after I had contacted him regarding FactionFriend updates for Legion and had taken the initiative to make fixes, for which he was quite grateful. I don't want to speak for him, but given that, I'm fairly confident that as long as we keep the legacy intact and credit the work he's done appropriately (and, should he ever come back, make his repo the single source of truth again and return ultimate control of the projects to him), he won't have an issue with our maintaining it -- so much so that [I've already started working on it](https://github.com/Dreamwalker-Collective/feed-o-matic/tree/classic) and will continue to try to reach him (I just sent him another message).

**Comment by datagutten** [comment]
Thank you for the information. I have made a working version which I published now: https://github.com/datagutten/feed-o-matic/tree/classic

**Comment by Tntdruid** [comment]
@fizzwidget ping 👍
